src/Stratis.Bitcoin.Features.Wallet/WalletSyncManager.cs

```diff
@@ -32,24 +32,27 @@ public class WalletSyncManager : IWalletSyncManager
 
         public ChainedHeader WalletTip => this.walletTip;
 
+        /// <summary>Queue which contains blocks that should be processed by <see cref="WalletManager"/></summary>
```

Comment:
Don't forget to end your sentences with a `.`

Reply:
ok